public/admin/config.yml

```
@@ -663,6 +663,25 @@ collections:
       - name: "location"
         label: "Location"
         widget: "string"
+      - name: "region"
+        label: "Region"
+        widget: "select"
+        options:
+          - label: "Australia/Oceania"
+            value: "australia-oceania"
+          - label: "North America"
+            value: "north-america"
+          - label: "South America"
+            value: "south-america"
+          - label: "Europe"
+            value: "europe"
+          - label: "APAC/Asia"
+            value: "apac-asia"
+          - label: "Africa"
+            value: "africa"
+          - label: "Middle East"
+            value: "middle-east"
```

Alphabetize, please.

```suggestion
          - label: "Africa"
           value: "africa"
          - label: "APAC/Asia" 
           value: "apac-asia"
          - label: "Australia/Oceania"
           value: "australia-oceania"
          - label: "Europe"
           value: "europe"
          - label: "Middle East"
           value: "middle-east"
          - label: "North America"
           value: "north-america"
          - label: "South America"
           value: "south-america"
```